data/json/recipes/recipe_others.json

```diff
@@ -3796,7 +3819,9 @@
   "difficulty" : 2,
```

Comment:
Actually no, it has nothing to do with your change, but I'll PR the change after I merge yours, which is otherwise good to go.